mod_wsbridge/mod_wsbridge.c

```diff
@@ -154,6 +184,8 @@ struct private_object {
 	struct lws_client_connect_info i;
 	char path[2048];
 	int state;
+	switch_bool_t audio_active;
+	switch_mutex_t *audio_active_mutex;
```

Comment:
Let's create a struct to couple the audio_active variable and its mutex, just like we did for the event Queue.  You already have `set_audio_active` and `get_audio_active` which are aligned with this approach.

```
struct AudioActive {
        switch_bool_t audio_active;
        switch_mutex_t *mutex;
};

void AudioActive_create(struct AudioActive *ac, switch_memory_pool_t *memory_pool) {
        ac->audio_active = TRUE;
        switch_mutex_init(&ac->mutex, SWITCH_MUTEX_NESTED, memory_pool);
}

void AudioActive_set(struct AudioActive *ac, switch_bool_t value) {
        switch_mutex_lock(ac->mutex);
        ac->audio_active = value;
        switch_mutex_unlock(ac->mutex);
}

switch_bool_t AudioActive_get(struct AudioActive *ac) {
        switch_bool_t value;
        switch_mutex_lock(ac->mutex);
        value = ac->audio_active;
        switch_mutex_unlock(ac->mutex);
        return value;
}

```